library/alloc/src/string.rs

```diff
@@ -139,6 +139,12 @@ use crate::vec::Vec;
 /// [`bytes`]: str::bytes
 /// [`chars`]: str::chars
 ///
+/// You can, however, index into a `String` in constant-time with anything that
+/// implements [`SliceIndex<str>`]. See the documentation for [`get`] for more details.
+///
+/// [`SliceIndex<str>`]: core::slice::SliceIndex
+/// [`get`]: str::get
```

Comment:
This takes them to something like https://doc.rust-lang.org/std/primitive.str.html#method.get. Is it possible to send them to https://doc.rust-lang.org/std/string/struct.String.html#method.get? That is, stay on the same page? I think I tried `String::get` here and got an error...

Reply:
If String::get doesn't work then it's probably not readily possible to link to it. I wouldn't worry about directing to a different page personally.